pyvista/core/filters/composite.py

```diff
@@ -154,3 +158,44 @@ def outline_corners(self, factor=0.2, nested=False, progress_bar=False):
             return DataSetFilters.outline_corners(self, factor=factor, progress_bar=progress_bar)
         box = pyvista.Box(bounds=self.bounds)
         return box.outline_corners(factor=factor, progress_bar=progress_bar)
+
+    def _compute_normals(
+        self,
+        cell_normals=True,
+        point_normals=True,
+        split_vertices=False,
+        flip_normals=False,
+        consistent_normals=True,
+        auto_orient_normals=False,
+        non_manifold_traversal=True,
+        feature_angle=30.0,
+        track_vertices=False,
+        progress_bar=False,
+    ):
+        """Compute point and/or cell normals for a multi-block dataset.
+
+        Warnings
+        --------
+        This will segmentation fault if a single block is non-polydata.
```

Comment:
Can we use `is_all_polydata` here to avoid a segfault?

Reply:
Implemented in 416ef55af169ce2da8d4443e3c1d17eeff9d7364.